### PR TITLE
bug fix + education models

### PR DESCRIPTION
**Pull request by pratzrao**

### CODE REVIEW

File: models/marts/education/education_marts_tests.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-version: 2
-
-models:
-  - name: education_scholarship_categories_agg
-    description: "Aggregates scholarship categories for educational purposes."
-    columns:
```
Comment (Ishankoradia) [inline models/marts/education/education_marts_tests.yml:6]: Are these test cases not valid any more ?

File: models/marts/wash/wash_marts_test.yml [hunk 2]
```diff
@@ -1,66 +0,0 @@
-version: 2
-
-models:
-  - name: wash_facilities
-    description: "Select all columns from facilities table as base data for wash facilities analysis"
-    columns:
```
Comment (Ishankoradia) [inline models/marts/wash/wash_marts_test.yml:6]: Same question here, why are we removing these tests ?

File: models/staging/education/staging_student_satisfaction_survey.sql [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH source_data AS (
+    SELECT
+        id,
+        data::jsonb AS json_data  -- Cast the data column to JSONB
```
Comment (Ishankoradia) [inline models/staging/education/staging_student_satisfaction_survey.sql:8]: Is this needed ? I am guessing this is from one of our custom connectors. 
As far as i remember, the data column should be a jsonb.

File: models/staging/education/staging_teacher_satisfaction_survey.sql [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH source_data AS (
+    SELECT
+        id,
+        data::jsonb AS json_data  -- Cast the data column to JSONB
```
Comment (Ishankoradia) [inline models/staging/education/staging_teacher_satisfaction_survey.sql:8]: Same question, do we really need this if the data column is already in jsonb format. 

I might be wrong too.

**Comment by Ishankoradia** [comment]
@pratzrao can you point out which bug are we fixing in this PR ? If you can tag the discord conversation or write a brief description would be great.

File: models/marts/education/education_marts_tests.yml (hunk 1 above)
Comment (pratzrao) [reply models/marts/education/education_marts_tests.yml:6]: Nope

File: models/marts/wash/wash_marts_test.yml (hunk 2 above)
Comment (pratzrao) [reply models/marts/wash/wash_marts_test.yml:6]: We restructured models, so they're irrelevant now

**Pull request merged**